web_embedding/element_embedding_demo/lib/main.dart

```diff
@@ -64,7 +64,8 @@ class _MyAppState extends State<MyApp> {
     return MaterialApp(
       title: 'Element embedding',
       theme: ThemeData(
-        primarySwatch: Colors.blue,
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.blue),
```

Comment:
Is this the same as `colorSchemeSeed: Colors.blue,`?

Reply:
Sounds like yes, according to the [`ThemeData` constructor docs](https://api.flutter.dev/flutter/material/ThemeData/ThemeData.html):

> When providing a [`ColorScheme`](https://api.flutter.dev/flutter/material/ColorScheme-class.html), apps can either provide one directly with the `colorScheme` parameter, or have one generated for them by using the `colorSchemeSeed` and `brightness` parameters. A generated color scheme will be based on the tones of `colorSchemeSeed` and all of its contrasting color will meet accessibility guidelines for readability. (See [`ColorScheme.fromSeed`](https://api.flutter.dev/flutter/material/ColorScheme/ColorScheme.fromSeed.html) for more details.)